theme-contract/src/main/java/com/orange/ods/theme/colors/OdsComponentsColors.kt

```diff
@@ -52,10 +68,40 @@ class OdsComponentsColors(
 }
 
 @Suppress("FunctionName")
+fun OdsComponentsColors(init: OdsComponentsColors.Builder.() -> Unit = {}): OdsComponentsColors.Builder {
+    return OdsComponentsColors.Builder().apply(init)
+}
+
+@Suppress("FunctionName")
+@Deprecated(
+    "Please use OdsComponentColors function with only init parameter instead.",
+    replaceWith = ReplaceWith("OdsComponentsColors(init)")
+)
 fun OdsComponentsColors(systemBarsBackground: Color, init: OdsComponentsColors.Builder.() -> Unit = {}): OdsComponentsColors.Builder {
-    return OdsComponentsColors.Builder(systemBarsBackground).apply(init)
+    return OdsComponentsColors.Builder().apply(init).apply {
+        statusBar = OdsStatusBarColors(background = systemBarsBackground)
+    }
 }
 
+/**
+ * Customizable colors for status bar
+ *
+ * @attribute background Background color of the status bar.
+ * @attribute isAppearanceLight Controls the items color of the status bar. When `true`, the items on the bar can be read clearly on a light background.
+ *
+ * Use [Colors.DefaultOdsStatusBarColors] to get the default component's colors based on Material Colors defined in theme
+ */
+data class OdsStatusBarColors(
+    val background: Color,
+    val isAppearanceLight: Boolean = false
```

Comment:
I set `isAppearanceLight` to `false` because this is the default appearance of a status bar in Material 2. Even if you are in light theme on your phone, ba default the app has a primary dark background status bar: [https://developer.android.com/develop/ui/views/theming/themes](https://developer.android.com/develop/ui/views/theming/themes) (This is maybe not the case in M3). There is an option to change this behavior to have a light status bar in the SDK but this is not the default behavior.